files/zh-cn/web/javascript/reference/global_objects/promise/index.md

```diff
@@ -5,76 +5,92 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise
 
 {{JSRef}}
 
-**`Promise`** 对象用于表示一个异步操作的最终完成（或失败）及其结果值。
+**`Promise`** 对象表示异步操作最终的完成（或失败）以及其结果值。
 
-{{AvailableInWorkers}}
-
-若想了解 promise 的工作方式以及如何使用它们，我们建议你先阅读[使用 promise](/zh-CN/docs/Web/JavaScript/Guide/Using_promises)。
+若想了解 promise 的工作方式以及如何使用它们，我们建议你先阅读[使用 Promise](/zh-CN/docs/Web/JavaScript/Guide/Using_promises)。
 
 ## 描述
 
-一个 `Promise` 对象代表一个在这个 promise 被创建出来时不一定已知值的代理。它让你能够把异步操作最终的成功返回值或者失败原因和相应的处理程序关联起来。这样使得异步方法可以像同步方法那样返回值：异步方法并不会立即返回最终的值，而是会返回一个 _promise_，以便在未来某个时候把值交给使用者。
+一个 **`Promise`** 是一个代理，它代表一个在创建 promise 时不一定已知的值。它允许你将处理程序与异步操作的最终成功值或失败原因关联起来。这使得异步方法可以像同步方法一样返回值：异步方法不会立即返回最终值，而是返回一个 _promise_，以便在将来的某个时间点提供该值。
 
 一个 `Promise` 必然处于以下几种状态之一：
 
 - *待定（pending）*：初始状态，既没有被兑现，也没有被拒绝。
 - *已兑现（fulfilled）*：意味着操作成功完成。
 - *已拒绝（rejected）*：意味着操作失败。
 
-待定状态的 Promise 对象要么会通过一个值*被兑现*，要么会通过一个原因（错误）*被拒绝*。当这些情况之一发生时，我们用 promise 的 `then` 方法排列起来的相关处理程序就会被调用。如果 promise 在一个相应的处理程序被绑定时就已经被兑现或被拒绝了，那么这个处理程序也同样会被调用，因此在完成异步操作和绑定处理方法之间不存在竞态条件。
+一个待定的 Promise *最终状态*可以是*已兑现*并返回一个 value，或者是*已拒绝*并返回一个 reason（错误）。当其中任意一种情况发生时，通过 Promise 的 `then` 方法串联的处理程序将被调用。如果绑定相应处理程序时 Promise 已经兑现或拒绝，这处理程序将被立即调用，因此在异步操作完成和绑定处理程序之间不存在竞态条件。
```

Comment:
why we don't translate 'value' and 'reason'?

Reply:
In the Promise.allSettled method returns an array of promise results - fulfilled return `{status: 'fulfilled', value: the value}` and rejected return `{status: 'rejected', reason: the reason}`. So I am considering not translating the `value` and `reason` because they have means of variables.

Reply:
Yes, the "reason" here would be _set_ as the `reason` key in the return object of `allSettled` (if we call the method). But this is a general description. We haven't given any object or function signature here. And the `reason` key should mean "reason", that is _why_ the promise is rejected. I don't think only using English here would bring more meaning here.

Reply:
Agree with that.